Deep_Learning/FeedFlash/backend/news_fetcher.py

```
@@ -0,0 +1,90 @@
+app_dir = os.path.join(current_dir, "..", "app")
+sys.path.append(app_dir)
+
+from summarizer import summarize_text
+
+API_KEY = os.getenv("NEWS_API")
+if not API_KEY:
+    raise ValueError("❌ NEWS_API is not set in environment variables")
+
+client = NewsApiClient(api_key=API_KEY)
+
+def fetch_news():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)